net/radius-mac/Makefile

```
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=radius-mac
+# Generally, our PKG_VERSION should match our PKG_SOURCE_VERSION
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/carlanton/radius-mac.git
+PKG_SOURCE_VERSION:=release-0.1
+PKG_MIRROR_HASH:=3d52453fd6c3aaef80aa6f0a65114721c38561a440a5165e42c864650a250cd3
+
+PKG_MAINTAINER:=Brian 'redbeard' Harrington <redbeard@dead-city.org>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/radius-mac
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=RADIUS server providing MAC addresss authentication
+	URL:=https://github.com/carlanton/radius-mac
+	DEPENDS:=+libuci
+endef
+
+define Package/radius-mac/description
+	A simple RADIUS server for MAC address authentication.
+endef
```

```suggestion
define Package/radius-mac
  SECTION:=net
  CATEGORY:=Network
  TITLE:=RADIUS server providing MAC address authentication
  URL:=https://github.com/carlanton/radius-mac
  DEPENDS:=+libuci
endef

define Package/radius-mac/description
  A simple RADIUS server for MAC address authentication.
endef
```

Just nitpicking - small typo and fix indentation for these two defines.